app/javascript/vue/item_relationships/RepositoryItemRelationshipsModal.vue

```diff
@@ -162,8 +163,11 @@ export default {
     };
   },
   computed: {
+    inventoryValueIsSelected() {
+      return this.selectedInventoryValue !== null;
```

Comment:
Is there a reason to not use this.selectedInventoryValue directly? As in `:disabled="!selectedInventoryValue"`